connector-schemas/websocket/table.json

```diff
@@ -11,14 +11,25 @@
             ],
             "format": "uri"
         },
-        "subscription_message": {
```

Comment:
This will break existing websocket connections in your DB. Do you think it's worth keeping the old "subscription_message" around to provide backwards-compatibility?